RadixWallet/Features/CreateAccount/Children/NameAccount/NameAccount+Reducer.swift

```diff
@@ -47,10 +47,15 @@ public struct NameAccount: Sendable, FeatureReducer {
 			return .none
 
 		case let .confirmNameButtonTapped(sanitizedName):
-			return .send(.delegate(.proceed(
+			return .run { _ in
+				await MainActor.run {
```

Comment:
Any reason to not use Focus to resign the responder?

Reply:
Seems to me it adds a lot of boilerplate (need to set the focus on the textfield, have the corresponding variable on the state and then also the action handling). It would be useful if we wanted to set the focus when opening the view or something, but if it is just to loose the focus I think this way is simpler.

Reply:
Agree, was just curious.
Could we then extract this in some separate utility that could be use to resign the responder? idk if it could be an extension on `Effect`

Reply:
Not sure about the context here don't don't be afraid to work outside TCA for things like focus, if you don't actually need to control it through a reducer. For UI only things at least, we are not TCA maximalists.